### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by teenaxta**
Added needed packages to run the gradio app as well as the provided code blocks

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,3 +11,6 @@ torch==1.7.1
 torchvision==0.8.2
 tqdm==4.65.0
 wandb==0.15.5
+transformers==4.28.0
```
Comment (piddnad) [inline requirements.txt:14]: It doesn't seem to require these two packages to run gradio app?

File: requirements.txt [hunk 2]
```diff
@@ -11,3 +11,6 @@ torch==1.7.1
...
+sentencepiece==0.1.99
```
Comment (piddnad) [inline requirements.txt:15]: Running the gradio application doesn't seem to require these two packages, so please consider removing them.

Comment (teenaxta) [reply requirements.txt]: ya i just just checked, indeed not needed. I wonder why i added them

**Comment by piddnad** [comment]
Fixed in #20 . Thank you still for your contribution. :)

**Pull request closed without merging**